db/post_migrate/20210528203310_post_migrate_gift_from_pm_to_pms.rb

```diff
@@ -0,0 +1,19 @@
+class PostMigrateGiftFromPmToPms < ActiveRecord::Migration[6.1]
+  def up
+    DiscourseAutomation::Field.where(name: 'giftee_assignment_message').each do |field|
```

Comment:
Any chances this migration can be done without using AR?

Reply:
done in https://github.com/discourse/discourse-automation/pull/21/commits/d05453fcda70b9470cbe4b8daa6f11b223e24fa5